docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md

```diff
@@ -1,6 +1,6 @@
 # Build cloud image with binary file
 
-Command line tools such as helm or kustomize are single binary files. We can package them into cloud images, and then install them by deploying cloud images on the master node.Let's take helm as an example to introduce how to package binary files into cloud image.
+Command line tools such as helm or kustomize are single binary files. Sealos can package them into cloud images, and then install them by deploying cloud images on the master node.Let's take helm as an example to introduce how to package binary files into cloud image.
```

Comment:
Sealos->sealos

Reply:
Command line tools such as helm and kustomize are single binary files. sealos can pack them into a cloud image, and then install those binaries by deploying cloud images on the master node. Let's take helm as an example to show you how to package binaries into cloud image.